packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -278,21 +278,21 @@ class _Fingerprint extends StatelessWidget {
           children: [
             if (hasPdcFingerprint && !haslocalFingerprint) ...[
               Text(
-                'Your PDC fingerprint has been detected. If you have local fingerprint, you can enter it below.',
+                'Your PDC fingerprint has been detected. If you have a local fingerprint, you can enter it below.',
                 style: theme.subtleTextStyle,
               ),
               const SizedBox(height: denseSpacing),
             ],
             if (isError) ...[
               const Text(
-                'Issue: no fingerprint detached locally or on PDC',
+                'Issue: no fingerprint detected locally or on PDC',
               ),
               const SizedBox(height: denseSpacing),
               const Text('Fix guide:'),
               const SizedBox(height: denseSpacing),
               Text(
                 'To fix this issue, release your app on Play Developer Console to get a fingerprint. '
-                'If you are not ready to release your app, enter a local fingerprint below can also allow you'
+                'If you are not ready to release your app, enter a local fingerprint below can also allow you '
                 'to proceed Android domain check.',
```

Comment:
"If you are not ready to release your app, you can proceed with the Android domain check by entering a local fingerprint below."